src/pal/inc/pal.h

```diff
@@ -5240,6 +5286,7 @@ PALIMPORT void * __cdecl memcpy(void *, const void *, size_t);
 #endif //defined(_DEBUG)
 PALIMPORT int    __cdecl memcmp(const void *, const void *, size_t);
 PALIMPORT void * __cdecl memset(void *, int, size_t);
+PALIMPORT wchar_t * __cdecl wmemset(wchar_t *, wchar_t, size_t);
```

Comment:
Any reason why we can't use `wmemset` on windows and the copy loop on everything else (with a bug tracking an eventual fix)?